pkg/log/status.go

```diff
@@ -147,57 +154,57 @@ func (s *Status) End(success bool) {
 // Namef will output the name of the component / application / project in a *bolded* manner
 func Namef(format string, a ...interface{}) {
 	bold := color.New(color.Bold).SprintFunc()
-	fmt.Printf("%s\n", bold(fmt.Sprintf(format, a...)))
+	fmt.Fprintf(GetStdout(), "%s\n", bold(fmt.Sprintf(format, a...)))
 }
 
 // Progressf will output in an appropriate "progress" manner
 func Progressf(format string, a ...interface{}) {
-	fmt.Printf("%s%s\n", prefixSpacing, fmt.Sprintf(format, a...))
+	fmt.Fprintf(GetStdout(), " %s%s\n", prefixSpacing, fmt.Sprintf(format, a...))
 }
 
 // Successf will output in an appropriate "progress" manner
 func Successf(format string, a ...interface{}) {
 	green := color.New(color.FgGreen).SprintFunc()
-	fmt.Printf("%s%s%s%s\n", prefixSpacing, green("✓"), suffixSpacing, fmt.Sprintf(format, a...))
+	fmt.Fprintf(GetStdout(), "%s%s%s%s\n", prefixSpacing, green(getSuccessString()), suffixSpacing, fmt.Sprintf(format, a...))
 }
 
 // Errorf will output in an appropriate "progress" manner
 func Errorf(format string, a ...interface{}) {
 	red := color.New(color.FgRed).SprintFunc()
-	fmt.Fprintf(os.Stderr, " %s%s%s\n", red("✗"), suffixSpacing, fmt.Sprintf(format, a...))
+	fmt.Fprintf(GetStdout(), " %s%s%s\n", red(getErrString()), suffixSpacing, fmt.Sprintf(format, a...))
```

Comment:
Would this not mean that error would not go to stderr but into stdout?

Reply:
Good catch, you're right. I'll fix that.